docs/src/content/docs/guides/pages.mdx

```diff
@@ -106,13 +106,35 @@ The following properties differ from Markdown frontmatter:
 - The [`editUrl`](/reference/frontmatter/#editurl) option requires a URL to display an edit link.
 - The [`sidebar`](/reference/frontmatter/#sidebar) property is not supported. In Markdown frontmatter, this option allows customization of [autogenerated link groups](/reference/configuration/#sidebar), which is not applicable to pages using the `<StarlightPage />` component.
```

Comment:
```suggestion
- The [`sidebar`](/reference/frontmatter/#sidebar) frontmatter property for customizing how the page appears in [autogenerated link groups](/reference/configuration/#sidebar) is not available. Pages using the `<StarlightPage />` component are not part of a collection and cannot be autogenerated in the sidebar.
```
OK, this might or might not be better, *but* I found it a little weird to read "the sidebar property is not supported" and then immediately see (admittedly, a different property) called "sidebar". 😄 

At the very least, that's why I threw "the sidebar FRONTMATTER property..." in there. Then, tried to distinguish *that* sidebar ("how the page is described in the sidebar") from the property below ("which sidebar to generate for the page")

Just take that how you will!

Reply:
Yeah, I can see that! Took the liberty of tweaking the very end of your suggestion so I could use it and give you credit 🙌